transactions/README.md

```diff
@@ -142,17 +141,96 @@ This schedule is conflict-serializable because there is an edge from transfer to
 
 ![serial schedule image](/transactions/cyclic.png)  
 
-## 2PL Permits Interleaved Access
+## 2 PL Only Allows Serializable Schedules
+The below example is precluded because Sum cannot grab the S-lock on B after it has released the S-lock on A.
+
+![serial schedule image](/transactions/2pl_example.png)  
 
-TODO: Summarize (preferably with ASCII art) L15 slides 37
+## 2PL Permits Interleaved Access
+Key rule is that once a transaction has released a lock, it is not allowed to obtain any other locks. Progression of time goes downward in the chart.
+
+```
+Transfer Transaction			Sum Transaction
+								ACQ S-Lock on A
+								Read A = 1
+ACQ S-Lock on A
+Read A = 1
+								ACQ S-Lock on B
+								Read B = 1
```

Comment:
Yeah, but that's just another interleaving thats possible? not sure what your point is